apps/database/src/models/Account.js

```diff
@@ -0,0 +1,39 @@
+import { Model, mergeSchemas, timestampsSchema } from '../util'
+import { Organization } from './Organization'
+import { User } from './User'
+
+export class Account extends Model {
+  static get tableName() {
+    return 'accounts'
+  }
+
+  static get jsonSchema() {
+    return mergeSchemas(timestampsSchema, {
+      properties: {
+        userId: { type: ['string', null] },
+        organizationId: { type: ['string', null] },
+      },
```

Comment:
one of with userId or organizationId null / required ?

`oneOf` is possible.

Reply:
I can't find a doc or an example about the `oneOf` options.